{"name": "test_get_user_valid", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_get_user.TestGetUser object at 0x0000025B1B984E10>\n\n    @allure.story(\"Get user details successfully\")\n    def test_get_user_valid(self):\n        api = UserAPI()\n        user = api.create_user(random_username(), random_email(), \"123456\").json()\n        user_id = user.get(\"id\") or user.get(\"data\", {}).get(\"id\", 1)\n        res = api.get_user(user_id)\n>       assert res.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\ntestcases\\test_get_user.py:15: AssertionError"}, "start": 1760525398474, "stop": 1760525401617, "uuid": "91112b2d-0060-4aa0-9217-a3ac50c9de9d", "historyId": "624b41f0d1259aa4a5ed9cb46af38b9f", "testCaseId": "624b41f0d1259aa4a5ed9cb46af38b9f", "fullName": "testcases.test_get_user.TestGetUser#test_get_user_valid", "labels": [{"name": "feature", "value": "Get User Details API"}, {"name": "story", "value": "Get user details successfully"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_get_user"}, {"name": "subSuite", "value": "TestGetUser"}, {"name": "host", "value": "DESKTOP-AC1JF6R"}, {"name": "thread", "value": "14528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_get_user"}], "titlePath": ["testcases", "test_get_user.py", "TestGetUser"]}